src/components/shared/storyImageCutout.js

```diff
@@ -0,0 +1,230 @@
+                                    <i className="fa-solid fa-play"></i> Watch Video<span className="visually-hidden">: {videoName}</span>
+                                </button>
+                            }
+                        /> : null }
+                     </Col>
+                     <Col lg={6} className="d-flex justify-content-center">
+                         <GatsbyImage image={getImage(foreground.src)} alt={foreground.alt} className="align-self-end img-fluid" />
+                     </Col>
+                 </Row>
+             </Container>
+         </Overlay.GatsbyImage>
+
+        {testimonial && 
+        <Testimonial className="d-flex justify-content-center">
+          <Container className="page-container">
+            <Row className="bg-transparent w-100 p-5 text-center">
+                <QuoteText className="display-3 text-white">
+                    <QuoteMark className="fad fa-quote-left pe-2" aria-hidden="true" /> 
+                        <em>{testimonial.field_story_quote}</em>
+                    <QuoteMark className="fad fa-quote-right ps-2" aria-hidden="true" />
+                </QuoteText>
+                {testimonial.field_quote_source && 
+                  <QuoteSource className="fs-3">~ {testimonial.field_quote_source}{testimonialQuoteSource}</QuoteSource>
+                }
+            </Row>
+          </Container>
+        </Testimonial>}
+      </div>
+    ) : null
+}
+
+export default StoryImageCutout
+
+export const query = graphql`
+  fragment StoryImageCutoutParagraphFragment on paragraph__story_image_cutout_background {
+    id
+    drupal_id
+    field_story_title
+    field_story_text {
+      processed
+    }
+    relationships {
+      field_story_image {
+        ... on media__image {
+          name
+          field_media_image {
+            alt
+          }
+          relationships {
+            field_media_image {
+              localFile {
+                publicURL
+                childImageSharp {
+                  gatsbyImageData(
+                    layout: CONSTRAINED,
+                    placeholder: BLURRED,
+                    height: 440,
+                  )
+                }
+              }
+            }
+          }
+        }
+      }
+
+      field_story_image_bg {
+        ... on media__image {
+          name
+          field_media_image {
+            alt
+          }
+          relationships {
+            field_media_image {
+              localFile {
+                publicURL
+                childImageSharp {
+                  gatsbyImageData(
+                    width: 1400,
+                    height: 500,
+                    placeholder: BLURRED, 
+                    layout: CONSTRAINED,
+                    transformOptions: {
+                      duotone: { highlight: "#000000", shadow: "#000000", opacity: 40 },
+                      cropFocus: NORTH,
```

Comment:
```suggestion
                      cropFocus: ENTROPY,
```

...because `NORTH` crops it to the top of an image. This works well for the Toronto skyline photo on https://mmtest.gatsbyjs.io/lang-bcomm/be-career-ready but not so well on https://mmtest.gatsbyjs.io/become-a-global-leader (where you completely lose the background image's focal point, i.e. the Arc de Triomphe)

Worth testing `ENTROPY` to see if it manages to crop according to the most complex or focal point of most photos.